src/main/java/teammates/ui/template/StudentCommentsFeedbackResponseCommentRow.java

```diff
@@ -0,0 +1,17 @@
+package teammates.ui.template;
+
+public class StudentCommentsFeedbackResponseCommentRow extends FeedbackResponseCommentRow {
```

Comment:
It's probably best not to bother with inheritance at the moment.
We can think about it after finishing this and scanning through the whole project for common points.